tests/assets/test_asset.py

```diff
@@ -98,18 +101,25 @@ def test_asset():
 
     purchase_account.request_tokens(10)
 
-    time.sleep(10)
-    logging.info(f'purchase_account {purchase_account.balance}')
-
+    time.sleep(2)
```

Comment:
_sleep()_: is this here just for testing purposes or is it suggesting more timing problems are lurking?